torch/testing/_internal/common_methods_invocations.py

```diff
@@ -9688,6 +9710,23 @@ def ref_pairwise_distance(input1, input2):
            sample_inputs_func=partial(sample_inputs_softmax_variant, with_dtype=True),
            assert_autodiffed=True,
            supports_out=False),
+    # `softmin` supports different dtypes based on whether `dtype` argument,
+    # is passed or not. Hence two OpInfo entries, one with dtype and other without.
+    OpInfo('nn.functional.softmin',
+           aten_name='softmin',
+           dtypesIfCPU=floating_types_and(torch.bfloat16),
+           dtypesIfCUDA=floating_types_and(torch.half, torch.bfloat16),
+           sample_inputs_func=sample_inputs_softmax_variant,
```

Comment:
Might as well expand the comment for this function, too:

https://github.com/pytorch/pytorch/blob/3cece9ffee74bef34bacbe2108dd4e91d2f4f5d5/torch/testing/_internal/common_methods_invocations.py#L5719